src/ModulusRemainder.h

```diff
@@ -40,6 +40,8 @@ ModulusRemainder modulus_remainder(Expr e, const Scope<ModulusRemainder> &scope)
  * to remainder if an answer could be found. */
 bool reduce_expr_modulo(Expr e, int modulus, int *remainder);
 
+bool reduce_expr_modulo(Expr e, int modulus, int *remainder, const Scope<ModulusRemainder> &scope);
```

Comment:
You can give this new variant the same doxygen comment as the other one by wrapping them with //@{ and //@} See IROperator.h for usage.